app/crud.py

```diff
@@ -438,6 +439,19 @@ def increment_proof_price_count(db: Session, proof: ProofFull):
     return proof
 
 
+def delete_proof(db: Session, db_proof: ProofFull) -> bool:
+    # we delete the image of the proof
+    file_path_obj = Path(db_proof.file_path)
+    # Check if the file exists
+    if file_path_obj.exists():
+        # remove the file
+        file_path_obj.unlink()
```

Comment:
@raphael0202 ok for you to hard delete proof images ?

Reply:
yes, no issue on my side :)